pkg/netdevice/netResourcePool_test.go

```diff
@@ -137,4 +147,55 @@ var _ = Describe("NetResourcePool", func() {
 			})
 		})
 	})
+	Describe("Saving and Cleaning DevInfo files ", func() {
+		Context("for valid pci devices", func() {
+			rc := &types.ResourceConfig{
+				ResourceName:   "fakeResource",
+				ResourcePrefix: "fakeOrg.io",
+				SelectorObj: &types.NetDeviceSelectors{
+					IsRdma: true,
+				},
+			}
+			nadutils := &mocks.NadUtils{}
+			nadutils.On("SaveDeviceInfoFile", "fakeOrg.io/fakeResource", "fake1", Anything).
+				Return(func(rName, id string, devInfo *nettypes.DeviceInfo) error {
+					if devInfo.Type != nettypes.DeviceInfoTypePCI || devInfo.Pci == nil || devInfo.Pci.PciAddress != "0000:01:00.1" {
+						return fmt.Errorf("wrong device info")
+					}
+					return nil
+				})
+			nadutils.On("SaveDeviceInfoFile", "fakeOrg.io/fakeResource", "fake2", Anything).
+				Return(func(rName, id string, devInfo *nettypes.DeviceInfo) error {
+					if devInfo.Type != nettypes.DeviceInfoTypePCI || devInfo.Pci == nil || devInfo.Pci.PciAddress != "0000:01:00.2" {
+						return fmt.Errorf("wrong device info")
+					}
+					return nil
+				})
+			nadutils.On("CleanDeviceInfoFile", "fakeOrg.io/fakeResource", "fake1").Return(nil)
```

Comment:
nit:  I would move nadutil mock initialization down to the `It` block and only mock whats needed per the specific test.
then you can use : nadutils.AssertExpectations(GinkgoT())

also perhaps add:
`t := GinkgoT()` in `Describe` block so `GinkgoT()` is not being called each time and use `t` where needed

Reply:
Done

Reply:
Done